<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="21577" started-at="2023-11-26T15:23:09 IST" finished-at="2023-11-26T15:23:30 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="21577" started-at="2023-11-26T15:23:09 IST" finished-at="2023-11-26T15:23:30 IST">
      <class name="Testcase.MakeMyTripSearchTestcase">
        <test-method status="PASS" signature="LaunchBrowsers()[pri:0, instance:Testcase.MakeMyTripSearchTestcase@a74868d]" name="LaunchBrowsers" is-config="true" duration-ms="5156" started-at="2023-11-26T15:23:03 IST" finished-at="2023-11-26T15:23:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowsers -->
        <test-method status="PASS" signature="CloseADDs()[pri:0, instance:Testcase.MakeMyTripSearchTestcase@a74868d]" name="CloseADDs" is-config="true" duration-ms="8380" started-at="2023-11-26T15:23:09 IST" finished-at="2023-11-26T15:23:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseADDs -->
        <test-method status="FAIL" signature="ValidSearch()[pri:0, instance:Testcase.MakeMyTripSearchTestcase@a74868d]" name="ValidSearch" duration-ms="13175" started-at="2023-11-26T15:23:17 IST" finished-at="2023-11-26T15:23:30 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Flights from Chennai to Pune] but found [Flights from Chennai to Bengaluru]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Flights from Chennai to Pune] but found [Flights from Chennai to Bengaluru]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Testcase.MakeMyTripSearchTestcase.ValidSearch(MakeMyTripSearchTestcase.java:81)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidSearch -->
        <test-method status="PASS" signature="quitTheBrowser()[pri:0, instance:Testcase.MakeMyTripSearchTestcase@a74868d]" name="quitTheBrowser" is-config="true" duration-ms="755" started-at="2023-11-26T15:23:30 IST" finished-at="2023-11-26T15:23:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitTheBrowser -->
      </class> <!-- Testcase.MakeMyTripSearchTestcase -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
